### DISCUSSION TITLE
base:ssl error when running salt-call state.highstate

### DISCUSSION THREAD

**Discussion by Leche671** [Security-Onion-Solutions/securityonion#3426] [Unsupported Versions]
I am experiencing this issue on a distributed deployment running 2.3.21. I tried running the "salt-call state.apply ca", but the "salt-call state.highstate" fails. 

I'm thinking this may be an issue in reference to the [Fix/salt ca ssl #1295](https://github.com/Security-Onion-Solutions/securityonion/pull/1295).  It doesn't say how I to resolve the error.

**Comment by weslambert** [comment]
@m0duspwnens , any thoughts?

**Reply by Leche671** [reply]
I verified that all the certificates and keys were present on the storage and sensor nodes and then disabled the SSL in top and all the nodes came up fine after a reboot.
Since it was an initial deployment I just reinstalled and reran setup on all the nodes and everything worked as expected.
